Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:alpine
+
+WORKDIR /src
+
+RUN apk update && apk add git cloc
+
+RUN sh -c "git clone https://github.com/benbalter/count-org-loc ." && \
```

Comment:
Curious the thinking behind using Git Clone here versus copying from the working directory?